packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -1577,7 +1577,7 @@ export default class RestApi {
       '/:collection(pools)/:object/vms',
       wrap(async (req, res) => {
         let srRef
-        const { sr } = req.params
```

Comment:
does it means that the SR parameter was always ommited since it's not defined in the route params ?

Reply:
Exactly